templates/create-daml-app/ui/src/components/LoginScreen.tsx.template

```diff
@@ -29,7 +29,7 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
       }
       onLogin(credentials);
     } catch(error) {
-      alert(`Unknown error:\n${JSON.stringify(error)}`);
```

Comment:
I’m a bit confused, what is the type of `error`? I.e., when does `JSON.stringify` return an empty object but string interpolation returns something useful?

Reply:
I'm also quite puzzled why one of them works and the other just returns '{}'.

Reply:
Ok checked, `error`'s type is `any`. Still not really sure how this works/does not work.

Reply:
It looks like you can overwrite `toJSON`. Not sure why you would do that to just return `{}` but if this helps, I’ll take it.